src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -968,46 +964,17 @@ def generate_PCR_7_default_config(self, filename):
     pal_high_frequency = pcr_1_test_config['palRecords'][0]['channelAssignment'][
         'primaryAssignment']['highFrequency']
 
-    # Load new device.
-    device_d = json.load(
-       open(os.path.join('testcases', 'testdata', 'device_d.json')))
+    user_id = pcr_1_test_config['registrationRequests'][0]['userId']
 
     # Make the PPA record consistent.
-    ppa_record, pal_record = makePpaAndPalRecordsConsistent(overlap_ppa_record,
-                                                            pcr_1_test_config['palRecords'],
-                                                            pal_low_frequency,
-                                                            pal_high_frequency,
-                                                            device_d['userId'])
-
-    # Place a device to be part of overlap PPA Zone.
-    device_d['installationParam']['latitude'], device_d['installationParam'][
-        'longitude'] = getRandomLatLongInPolygon(ppa_record)
-
-    # Set the AntennaGain and EIRP capability.
-    device_d['installationParam']['eirpCapability'] = 47
-    device_d['installationParam']['antennaGain'] = 16
-
-    # Set the conditionals for new device.
-    conditionals_d = {
-        'cbsdCategory': device_d['cbsdCategory'],
-        'fccId': device_d['fccId'],
-        'cbsdSerialNumber': device_d['cbsdSerialNumber'],
-        'airInterface': device_d['airInterface'],
-        'installationParam': device_d['installationParam'],
-        'measCapability': device_d['measCapability']
-    }
-    conditionals = [conditionals_d]
-    del device_d['installationParam']
-    del device_d['cbsdCategory']
-    del device_d['airInterface']
-    del device_d['measCapability']
-
+    overlap_ppa_record, pal_record = makePpaAndPalRecordsConsistent(ppa_record,
```

Comment:
Are you sure you can do this? I think that `makePpaAndPalRecordsConsistent` will fail when it calls `makePalRecordsConsistent` and the PAL records are missing the `fipsCode` and `censusYear` keys (which are removed in the same function).

Reply:
Thanks for catching this error. You are right . Calling makePpaAndPalRecordsConsistent  is no longer required and simply loaded the ppa contour to overlapping_ppa_record to achieve the expected outcome of this test case.

Reply:
I think you still need to e.g. add the right PAL IDs to the PPA record. As is, the SAS UUT would not know which frequency range to use for the PPA.

Reply:
Based on the SSC WG5 PAL Database Specification WINNF-TS-0245-V1.0.0  6.1.2 LicenseInformation object  
we retrieved the fispCode and censusYear from licenseAreaExtent field in pal_records generated part of PCR 1 test which has format
“zone/census_tract/census/$YEAR/$FIPS”  and update back to pal_records 
invoke makePpaAndPalRecordsConsistent by passing modified pal_records so that this method will update necessary Information required to make 
PPA record consistent.